prometheus/prometheus_targets.yml

```diff
@@ -12,8 +12,8 @@
     - query-runner:6060
     - repo-updater:6060
     - syntect-server:6060
-    - lsif-server:3186
-    - lsif-server:3187
+    - precise-code-intel-api-server:3186
+    - precise-code-intel-bundle-manager:3187
```

Comment:
Where is the entry or the worker?

Reply:
It's federated in prometheus as it can run more than one metrics endpoint.